crates/uv-cli/src/lib.rs

```diff
@@ -3135,6 +3135,33 @@ pub struct SyncArgs {
     #[arg(long, conflicts_with = "all_packages")]
     pub package: Option<PackageName>,
 
+    /// Sync the virtual environment for the specified PEP 723 Python script, rather than the current
+    /// project.
```

Comment:
Minor preference to keep this on one line for brevity / readability in the CLI short help.